hrjobcontract/js/src/job-contract/controllers/modal/modal-contract-new.controller.js

```diff
@@ -316,6 +313,16 @@ define([
         $scope.entity.leave[index].add_public_holidays = utils.absenceTypes[absenceTypeID].add_public_holiday_to_entitlement;
       });
     }
+
+    /*
+     * Set the entity object
```

Comment:
honestly i don't know if this warrants a separate function at all. just put this two lines directly in the `init()` function